CHANGELOG.md

```diff
@@ -10,12 +10,318 @@ New features:
 
 Bugfixes:
 
-* Only include direct dependencies in the output for `purs graph` (#3993, @colinwahl)
+Other improvements:
 
-Fixes a bug where the transitive closure of a module's dependencies
-where included in the `depends` field in the output for `purs graph`.
+## v0.14.0
 
-Other improvements:
+### Polykinds
+
+Polymorphic kinds, based on the [Kind Inference for Datatypes](https://richarde.dev/papers/2020/kind-inference/kind-inference.pdf) paper (#3779, #3831, #3929, @natefaubion)
```

Comment:
It may be useful to add an explanation or demonstration of why this feature is useful (perhaps @natefaubion has ideas?). For example, my (admittedly incomplete and limited) understanding is that we now have kind polymorphism + kind inference, which enables things like a single `Proxy` type instead of the many currently used for different kinds.

Can we explain what makes this feature a noteworthy addition to the language in a few sentences and/or a brief example?

Reply:
I mean, a single `Proxy` type is descriptive in itself. If one wanted to define their own kind, they needed to define their custom `Proxy` type, too. It's this whole "each kind must have it's own X" problem that polykinds solves. For example, we don't need a kind-specific type-level list anymore. We can have a kind-generic type-level list now.
I don't think that's all of polykinds, but it is a significant part.

Reply:
I agree that describing the single `Proxy` type is enough of a motivating example on its own. I think we should say that explicitly in this section, briefly, to motivate why this feature is a useful addition to the language.

Reply:
What do you think of the short explanation I added?

Reply:
I think it's great! 👍